src/dependability.md

```diff
@@ -80,8 +80,10 @@ Destructors are executed while panicking, and in that context a failing
 destructor causes the program to abort.
 
 Instead of failing in a destructor, provide a separate method for checking for
-clean teardown, e.g. a `close` method, that returns a `Result` to signal
-problems.
+clean teardown, e.g. a `close` method, that consumes `self` and returns a
```

Comment:
I don't think "consumes self" for close methods is a consensus position.

Previous discussion: https://github.com/rust-lang/api-guidelines/discussions/61

Reply:
I had not realized this was something that had been discussed before, or that there was a lack of consensus on it. Since that isn't the point of this PR, I'll remove it.